### DISCUSSION TITLE
Send commands to a running Helix process from command line

### DISCUSSION THREAD

**Discussion by rkshthrmsh** [helix-editor/helix#6402] [General] [answered]
I am attempting to automate some processes in Helix and wanted to understand if there is a way to send commands to a running hx process from the command line. For instance, reloading the config for an open hx session from the terminal.

**Comment by the-mikedavis** [comment, marked as answer]
This is not currently possible and falls under https://github.com/helix-editor/helix/issues/312. However, you can send SIGUSR1 to a Helix process to tell it to reload the configuration from disk. (See the bottom of the section here: https://docs.helix-editor.com/master/configuration.html?highlight=signal#configuration)

**Reply by gabydd** [reply]
Just had the thought you could use this to: first change a helix keybind that holds the command you want to send to helix, then tell helix to reload the config and now you just press the key and get the command sent to you.

**Reply by rkshthrmsh** [reply]
Thanks @the-mikedavis, that solution works for my use case.
